### DISCUSSION TITLE
Using code-server, is there a way to always re-open the workspace with the same file opened ?

### DISCUSSION THREAD

**Discussion by galesky** [coder/code-server#4314] [Q&A]
What I'm trying to do is that every time I open `code-server` on the Browser, I see the same folder with an open `README.md` file.

I could open the `folder` like this
```
http://coder-instance:5050/?folder=/home/coder/project
```
Or even the `workspace`
```
http://coder-instance:5050/?workspace=/home/coder/otherpproject/workspace.code-workspace
```
**Doing the same for a file would be awesome.**

---

I thought of messing with the history files on 
```
~/.local/share/code-server/User/state
```
But I think this could be very fragile.

---

**I there a way to achieve this ?**

Extra info: This instance is running on a K8s cluster

**Comment by jsjoeio** [comment]
I don't think this is currently supported but we do have an issue tracking it [here](https://github.com/cdr/code-server/issues/1964).

I agree, this would be super handy!

cc @bpmct @code-asher
